materials_helper_functions.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np 
+import pandas as pd 
+
+def total_molecular_mass_per_molecule_dict(data,tankID,WastePhase,compounds,molecules):
+    c = 12.011
+    h = 1.008
+    o = 15.999
+    p = 30.973761998
+    n = 14.007
+    cl = 35.45
+    f = 18.998403162
+    s = 32.06
+    tl = 204.38
```

Comment:
You should use the atomic weight function from openmc to get these rather than specifying them yourself

Reply:
Thanks. Just updated it